css/jplayer-flat-audio-theme.css

```diff
@@ -172,4 +172,38 @@
 .jp-state-no-volume .jp-progress,
 .jp-state-no-volume .jp-time-holder {
   right: 0;
+}
+
+/* Audio Playlist */
```

Comment:
None of these classes are used in anywhere in our samples (not in the README, nor in index.html).

Perhaps there should be another page on the preview site, playlist.html, that can be linked from index.html and would show a similar sample with a playlist. Then, the README could briefly mention the playlist capability and say something like, "see playlist.html for a functional example".